static-assets/css/style.css

```diff
@@ -1111,3 +1120,20 @@ button.btn.btn-default {
 .components-container:empty {
   padding: 50px;
 }
+
+
+/* ICE mode */
+
+.craftercms-ice-on #trainer .trainer-overlay {
```

Comment:
Make sure hovering doesn't trigger any effects

Reply:
Tested and it doesn't. The important rule in transform was added to avoid the scaling when hovering, and no other effects are being triggered.